NIPTool/server/views.py

```diff
@@ -342,3 +354,34 @@ def update():
                 )
 
     return redirect(request.referrer)
+
+
+@server_bp.route("/load", methods=["POST"])
+def load():
+    """Function to load data into the database with rest"""
+
+    request_data = request.form
+    v = Validator(batch_load_schema)
+    if not v.validate(request_data):
+        return "Incomplete batch load request", 400
```

Comment:
Have you tested this? I would rather handle it by returning a json object. Like this:

http://shzhangji.com/blog/2018/04/07/error-handling-in-restful-api/

You don't have to use a BadRequest class, but I think you should return a json object as in the example